doc_admin/AdminManual.adoc

```diff
@@ -344,11 +344,16 @@ archive remain unchanged once extracted, otherwise you will not be able to start
 appropriate manner.
 
 NOTE: It is important to understand that the portable version is meant to be a trial, that is, to have a look at the software
-or used in a very small facility (like a dispensary) with only one computer.  If the facility is large and needs a networked
+or used in a very small facility (like a dispensary) with only one computer. If the facility is large and needs a networked
 architecture, the full installation version is recommended.
 
+include::https://raw.githubusercontent.com/informatici/openhospital/master/POH-README.md[lines=21..197,leveloffset=+2]
+
+NOTE: Folder Structure and Known Issues at 
```

Comment:
Appears to have capitalization that isn't needed; I would suggest `NOTE: Folder structure and known issues are found at:`

Reply:
Thanks!